test/memcached-connections.test.js

```diff
@@ -166,18 +166,10 @@ describe('Memcached connections', function () {
           assert.throws(function() { throw err }, /connect ECONNREFUSED/);
             // Third request should find no servers
             memcached.get('idontcare', function(err) {
-            assert.throws(function() { throw err }, /not available/);
-              // Give enough time for server to reconnect
-              setTimeout(function() {
-                // Server should be reconnected, but expect ECONNREFUSED
-                memcached.get('idontcare', function(err) {
-                  assert.throws(function() { throw err }, /connect ECONNREFUSED/);
-                  assert.deepEqual(memcached.issues[server].failures,
-                    memcached.issues[server].config.failures);
-                  memcached.end();
-                  done();
-                });
-              }, 150);
+              assert.throws(function() { throw err }, /not available/);
```

Comment:
This may change the original idea of this test case (should reset failures after reconnecting to failed server), because it will not reconnect successfully.